metricbeat/module/redis/info/info.go

```diff
@@ -172,27 +178,27 @@ func (m *MetricSeter) Setup(ms *helper.MetricSet) error {
 func (m *MetricSeter) Fetch(ms *helper.MetricSet) (events []common.MapStr, err error) {
 
 	for _, host := range ms.Config.Hosts {
-		c := m.redisPools[host].Get()
-
-		// TODO: Do better error reporting
-		if c == nil {
```

Comment:
This part was removed in the PR. You mentioned earlier that this check should happen with ping()?